openlibrary/accounts/model.py

```diff
@@ -940,6 +986,17 @@ def audit_accounts(
         }
         ol_account.save_s3_keys(s3_keys)
 
+    # Handle Print Disability Processing
+    has_special_access = getattr(ia_account, 'has_disability_access', False)
+    if pda := web.cookies().get("pda") or ol_account.pd_authority:
+        if has_special_access:
+            ol_account.update_pd(pda, PDRequestStatus.FULFILLED.value)
+        elif not pda:
+            ol_account.update_pd(pda, PDRequestStatus.REQUESTED.value)
+        if ol_account.pd_status == PDRequestStatus.REQUESTED.value:
+            ol_account.send_pd_email()
+            ol_account.update_pd(pda, PDRequestStatus.EMAILED.value)
```

Comment:
This needs the most scrutiny; 

When a patron first **registers** an account, there is no `audit` performed. The only thing that happens is a `pda` cookie is set.

When they log in, an audit performed and (I think) what we want to happen is:
1. We notice there's a `pda` cookie
2. We set their `rpd` status to RECEIVED (unless the account is somehow already FULFILLED) -> We then send an email if needed and set rpd again to emailed.